tools/manticore-analysis/analysis.md

```diff
@@ -0,0 +1,56 @@
+# Manticore
+[Manticore](https://github.com/trailofbits/manticore) is a versatile symbolic execution tool that can analyze Ethereum Smart Contracts,
+Linux binaries, and Windows binaries. It's widely used for security auditing and testing of
+applications where security is paramount. Its main features include:
+- **Program Exploration**: executing a program with symbolic inputs and exploring all the possible states it can reach,
+- **Input Generation**: producing concrete inputs that result in a given program state,
+- **Error Discovery**: detecting crashes and other failure cases in binaries and Smart Contracts,
+- **Instrumentation**: providing fine-grained control of state exploration via event callbacks and instruction hooks,
+- **Programmatic Interface**: exposing programmatic access to its analysis engine via a Python API.
+
+Manticore can analyze the following types of programs:
+
+- Ethereum Smart Contracts (EVM bytecode)
+- Linux ELF binaries (x86, x86_64, aarch64, and ARMv7)
+- WASM Modules
+
+## Installation and execution:
+- Installing by PIP results in [protobuf incorrect version error](sample_execution/execution_errors.md#pip-instalation)
+  (described better here: https://github.com/trailofbits/manticore/issues/2600)
+- Build attempt with docker image version 3.7+ result in [attribute error](sample_execution/execution_errors.md#docker-v37-and-latest-)
+- Build attempt with docker image version 3.6 results in [tool custom exception](sample_execution/execution_errors.md#docker-v36)
```

Comment:
I don't think we should have this in installation execution, maybe as a  side note. since is not relevant for execution of the tool.